pkg/apis/openstackproviderconfig/v1alpha1/types.go

```diff
@@ -69,6 +69,13 @@ type OpenstackProviderSpec struct {
 	Trunk bool `json:"trunk,omitempty"`
 
 	RootVolume RootVolume `json:"root_volume,omitempty"`
+
+	// Server tags
+	// Requires Nova api 2.52 minimum!
+	Tags []string `json:"tags,omitempty"`
+
+	// Metadata mapping. Allows you to create a map of key value pairs to add to the server instance.
+	ServerMetadata map[string]string `json:"server_metadata,omitempty"`
```

Comment:
Rename to `json:"serverMetadata,omitempty"` to be consistent.

Reply:
yup, on it :D

Reply:
done